keras_retinanet/bin/train.py

```diff
@@ -467,17 +501,24 @@ def main(args=None):
         prediction_model,
         validation_generator,
         args,
+        #tensorboard_image=writer,
     )
 
     # start training
+
+        
     training_model.fit_generator(
         generator=train_generator,
         steps_per_epoch=args.steps,
+        validation_data=validation_generator if args.val_loss else None,
+        validation_steps= int(round(validation_generator.size()/args.batch_size)) if args.val_loss else None,
```

Comment:
This line will probably complain with flake8, because there are no whitespaces around the divide operator.